src/components/AdminNavbar/AdminNavbar.tsx

```diff
@@ -190,11 +190,7 @@ function AdminNavbar({ targets, url_1 }: NavbarProps): JSX.Element {
                 data-testid="logoutDropdown"
               >
                 {data?.organizations[0].image ? (
-                  <img
-                    src={data?.organizations[0].image}
```

Comment:
1. Using "More" now requires translations.
2. Can you think of a way in which we can use a image that is suitable for this operation that could be used?

Reply:
I did a quick research and the most suitable representation I could find are icons, I couldn't find an image that could fit into our design and theme, if we must use an image it may have to be custom made for this project, my only problem with replacing "More" with an icon is that we already have an icon there, an alternative is that we remove "More" and increase the size of our existing icon.
![icons](https://user-images.githubusercontent.com/104039356/226124510-06400f53-de2d-4639-ad6d-2ea08f68aef8.jpg)

Reply:
What about a hamburger icon?
![image](https://user-images.githubusercontent.com/16875803/226125721-286e0e56-de98-4d50-86e0-f5a2ed9d0407.png)

Reply:
We would have a problem of duplicate icons displaying on screens <=1199, will this be acceptable?

![ham-burger](https://user-images.githubusercontent.com/104039356/226125941-11b56556-4448-48ed-9d58-096f5db9c7d7.jpg)

Reply:
I think it might look like an error.

Reply:
But it would have been there before, so the user would know the functionality. As you seem to be online let's continue this on slack.

Reply:
Okay, ill send you a message now.